README.jp.md

```diff
@@ -0,0 +1,803 @@
+
+#### デバイス接続での自動起動
+
+[AutoAdb]を使用可能です:
+
+```bash
+autoadb scrcpy -s '{}'
+```
+
+[AutoAdb]: https://github.com/rom1v/autoadb
+
+#### SSHトンネル
+
+リモートデバイスに接続するため、ローカル`adb`クライアントからリモート`adb`サーバーへ
+接続することが可能です(同じバージョンの _adb_ プロトコルを使用している場合):
+
+```bash
+adb kill-server    # 5037ポートのローカルadbサーバーを終了する
+ssh -CN -L5037:localhost:5037 -R27183:localhost:27183 your_remote_computer
+# オープンしたままにする
+```
+
+他の端末から:
+
+```bash
+scrcpy
+```
+
+リモートポート転送の有効化を回避するためには、代わりに転送接続を
+強制することができます(`-R`の代わりに`-L`を使用することに注意):
+
+```bash
+adb kill-server    # 5037ポートのローカルadbサーバーを終了する
+ssh -CN -L5037:localhost:5037 -L27183:localhost:27183 your_remote_computer
+# オープンしたままにする
+```
+
+他の端末から:
+
+```bash
+scrcpy --force-adb-forward
+```
+
+
+ワイヤレス接続と同様に、クオリティを下げると便利な場合があります:
+
+```
+scrcpy -b2M -m800 --max-fps 15
+```
+
+### ウィンドウ構成
+
+#### タイトル
+
+ウィンドウのタイトルはデバイスモデルが初期値です。これは変更できます:
+
+```bash
+scrcpy --window-title 'My device'
+```
+
+#### 位置とサイズ
+
+ウィンドウの位置とサイズの初期値を指定できます:
+
+```bash
+scrcpy --window-x 100 --window-y 100 --window-width 800 --window-height 600
+```
+
+#### ボーダーレス
+
+ウィンドウの装飾を無効化するには:
+
+```bash
+scrcpy --window-borderless
+```
+
+#### 常に画面のトップ
+
+scrcpyの画面を常にトップにするには:
+
+```bash
+scrcpy --always-on-top
+```
+
+#### フルスクリーン
+
+アプリケーションを直接フルスクリーンで開始できます:
+
+```bash
+scrcpy --fullscreen
+scrcpy -f  # 短縮版
+```
+
+フルスクリーンは、次のコマンドで動的に切り替えることができます <kbd>MOD</kbd>+<kbd>f</kbd>
+
+
+#### 回転
+
+ウィンドウは回転することができます:
+
+```bash
+scrcpy --rotation 1
+```
+
+設定可能な値:
+ - `0`: 回転なし
+ - `1`: 90° 反時計回り
+ - `2`: 180°
+ - `3`: 90° 時計回り
+
+回転は次のコマンドで動的に変更することができます。 <kbd>MOD</kbd>+<kbd>←</kbd>
+_(左)_ 、 <kbd>MOD</kbd>+<kbd>→</kbd> _(右)_
+
+_scrcpy_ は3つの回転を管理することに注意:
+ - <kbd>MOD</kbd>+<kbd>r</kbd>はデバイスに縦向きと横向きの切り替えを要求する
+   (現在実行中のアプリで要求している向きをサポートしていない場合、拒否することがある)
+ - [`--lock-video-orientation`](#ビデオの向きをロックする)は、ミラーリングする向きを
+    変更する(デバイスからPCへ送信される向き)。録画に影響します。
+ - `--rotation` (もしくは<kbd>MOD</kbd>+<kbd>←</kbd>/<kbd>MOD</kbd>+<kbd>→</kbd>)
+    は、ウィンドウのコンテンツのみを回転します。これは表示にのみに影響し、録画
+    には影響しません。
+
+### 他のミラーリングオプション
+
+#### Read-only　リードオンリー
+
+制御を無効にするには(デバイスと対話する全てのもの:入力キー、マウスイベント、ファイル
+のドラッグ&ドロップ):
+
+```bash
+scrcpy --no-control
+scrcpy -n
+```
+
+#### ディスプレイ
+
+いくつか利用可能なディスプレイがある場合、ミラーリングするディスプレイを選択できます:
+
+```bash
+scrcpy --display 1
+```
+
+ディスプレイIDのリストは次の方法で取得できます:
+
+```
+adb shell dumpsys display   # search "mDisplayId=" in the output
+```
+
+セカンダリディスプレイは、デバイスが少なくともAndroid 10の場合にコントロール可能です。
+(それ以外ではリードオンリーでミラーリングされます)
+
+
+#### 起動状態にする
+
+デバイス接続時、少し遅れてからデバイスのスリープを防ぐには:
+
+```bash
+scrcpy --stay-awake
+scrcpy -w
+```
+
+scrcpyが閉じられた時、初期状態に復元されます。
+
+#### 画面OFF
+
+コマンドラインオプションを使用することで、ミラーリングの開始時にデバイスの画面を
+OFFにすることができます:
+
+```bash
+scrcpy --turn-screen-off
+scrcpy -S
+```
+
+もしくは、<kbd>MOD</kbd>+<kbd>o</kbd>を押すことでいつでもできます。
+
+元に戻すには、<kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>o</kbd>を押します。
+
+Androidでは、`POWER`ボタンはいつでも画面を表示します。
+便宜上、`POWER`がscrcpyを介して(右クリックもしくは<kbd>MOD</kbd>+<kbd>p</kbd>を
+介して)送信される場合、(ベストエフォートベースで)少し遅れて、強制的に画面を
+非表示にします。
+物理的な`POWER`ボタンを押しても画面は表示されます。
```

Comment:
押しても -> なら?

Reply:
I'm sorry. I couldn't get your suggestion

Reply:
Normally, the physical POWER button will turn on the screen, right?

In scrcpy, it will turn off screen after injecting a POWER event. But pressing the physical POWER button still turn the screen on. 

So there shouldn't be "ても"

Reply:
Oh... Now I understand the detail.
Thank you.